bandit/formatters/html.py

```diff
@@ -263,6 +263,7 @@ def report(manager, fileobj, sev_level, conf_level, lines=-1):
     <b>Severity: </b>{severity}<br>
     <b>Confidence: </b>{confidence}<br>
     <b>File: </b><a href="{path}" target="_blank">{path}</a> <br>
+    <b>Line number: </b>{line_number}</a> <br>
     <b>More info: </b><a href="{url}" target="_blank">{url}</a><br>
```

Comment:
How about also adding the column number?  Thanks

Reply:
No problem! I'll add that change in another PR.